docs/content/stable/deploy/kubernetes/best-practices.md

```diff
@@ -63,6 +56,6 @@ The following table summarizes the features and when to use local or remote stor
   </tr>
 </table>
 
+While local storage offers higher throughput and lower latency at a lower cost, it comes with significant limitations. Maintenance operations like cluster node pool upgrades, rolling operations to upgrade software or set flags can cause the pod to lose its local copy and require a full remote bootstrap of all local tablets from its peers. This can be time consuming for large databases. Further, for large clusters, there is always a non-zero risk that there is another pod movement during such maintenance operations causing data loss for a subset of tablets. Therefore, we do not recommend using local storage for most production use cases.
```

Comment:
```suggestion
While local storage offers higher throughput and lower latency at a lower cost, it comes with significant limitations. Maintenance operations like cluster node pool upgrades, or rolling operations to upgrade software or set flags can cause a pod to lose its local copy and require a full remote bootstrap of all local tablets from its peers. This can be time consuming for large databases. Further, for large clusters, there is always a non-zero risk of another pod movement happening during these maintenance operations, which would cause data loss for a subset of tablets. Therefore, using local storage is not recommended for most production use cases.
```